inst/stan/functions/calc_re_parametres.stan

```diff
@@ -0,0 +1,10 @@
+array[] real calc_re_parameters(array[] real lambda, vector nb_size, int model_obs) {
```

Comment:
Missing documentation

Reply:
i.e here you would write I added documentation in <commit-hash> and then resolve it.

Reply:
@barbora-sobolova for awareness